types/puppeteer/index.d.ts

```diff
@@ -11,6 +11,18 @@
 import { EventEmitter } from "events";
 import { ChildProcess } from "child_process";
 
+// Stub DOM types when the dom lib isn't included
```

Comment:
Unrelated changes, whoops.

Reply:
If it's not needed could you make another PR removing it?

Reply:
I'll update this PR in a bit